src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/RealtimeBlurView.Android.cs

```diff
@@ -0,0 +1,413 @@
+﻿/*
+
+Implementation based on https://github.com/mmin18/RealtimeBlurView.
+with some modifications and removal of unused features.
+
+------------------------------------------------------------------------------
+
+ https://github.com/mmin18/RealtimeBlurView
+ Latest commit    82df352     on 24 May 2019
+
+ Copyright 2016 Tu Yimin (http://github.com/mmin18)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+------------------------------------------------------------------------------
+ Adapted to csharp by Jean-Marie Alfonsi
+------------------------------------------------------------------------------
+*/
+
+using System;
+using System.Diagnostics;
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+
+namespace Uno.UI.Xaml.Media
+{
+	internal partial class RealtimeBlurView : View
+	{
+		private float mDownsampleFactor = 4;
+		private int mOverlayColor; // default #aaffffff
+		private float mBlurRadius = 10; // (0 < r <= 25)
+
+		private IBlurImpl mBlurImpl;
+		private bool mDirty;
+		private Bitmap mBitmapToBlur, mBlurredBitmap;
+		private Canvas mBlurringCanvas;
+		private bool mIsRendering;
+		private Paint mPaint;
+		private readonly Rect mRectSrc = new Rect();
+		private readonly Rect mRectDst = new Rect();
+		// mDecorView should be the root view of the activity (even if you are on a different window like a dialog)
+		private View mDecorView;
+		// If the view is on different root view (usually means we are on a PopupWindow),
+		// we need to manually call invalidate() in onPreDraw(), otherwise we will not be able to see the changes
+		private bool mDifferentRoot;
+		private static int RENDERING_COUNT;
+
+		private readonly PreDrawListener _preDrawListener;
+
+		public RealtimeBlurView(Context context) : base(context)
+		{
+			_preDrawListener = new PreDrawListener(this);
+			mBlurImpl = GetBlurImplementation(); // provide your own by override getBlurImpl()			
+			mBlurRadius = TypedValue.ApplyDimension(ComplexUnitType.Dip, 10, context.Resources.DisplayMetrics);
+			//mDownsampleFactor = a.getFloat(R.styleable.RealtimeBlurView_realtimeDownsampleFactor, 4);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)